forms/api/service.php

```
@@ -0,0 +1,590 @@
+
+    return $total;
+}
+
+function create_update_order(
+    string $form_id,
+    string $submission_id,
+    array $selected_addon_ids,
+    EventType $event_type,
+    MealType $meal_type,
+    Currency $currency = Currency::PEN,
+    ?string $existing_order_id,
```

PHPStan (2.1.46)

[error] Deprecated in PHP 8.0: Required parameter $existing_order_id follows optional parameter $currency.


(parameter.requiredAfterOptional)